config/bollmaker.yaml

```diff
@@ -56,6 +56,19 @@ exchangeStrategies:
     # For short position, you will only place buy order below the price (= average cost * (1 - minProfitSpread))
     minProfitSpread: 0.1%
 
+    # EXPERIMENTAL
```

Comment:
can you add a block comment for EXPERIMENTAL? or we might think all the following ocnfig are experimental.